tests/common.py

```
@@ -0,0 +1,45 @@
+def start_redis():
+    global redis_server, client, g, shutdown
+    port = os.environ.get("PORT", "6379")
+    target = os.environ.get("TARGET",
+                            "target/debug/libfalkordb.dylib" if platform.system() == "Darwin" else "target/debug/libfalkordb.so")
+    r = Redis(port=port)
+    try:
+        r.ping()
+        client = FalkorDB(port=port)
+        g = client.select_graph("test")
+        return
+    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)